### DISCUSSION TITLE
C-number in summary method

### DISCUSSION THREAD

**Discussion by NantiaL** [opencobra/cobrapy#1100] [Q&A]
### Checklist

- [X] I searched the [documentation](https://cobrapy.readthedocs.io).
- [X] I looked through [similar issues on GitHub](https://github.com/opencobra/cobrapy/issues).
- [X] I looked up "How to do ... in cobrapy" on a search engine.

### Question
Hello, 
I was wondering what the C-number in the model.summary() indicates. 

Thanks in advance!

**Comment by Midnighter** [comment]
Hey, I'm trying to make use of the new discussion feature. I hope you don't mind my moving your question here.

The summary lists the number of Carbon atoms in the metabolite since that is often the limiting resource. It would be straight forward to enable you to list other atoms, too, if that is a desirable feature.

**Reply by NantiaL** [reply]
Hi, thank you very much for your response. In which terms you say " a limiting factor"?

**Reply by Midnighter** [reply]
In the context of metabolic engineering and in minimal media, the growth rate or production flux typically directly depend on the carbon source uptake rate.

**Reply by NantiaL** [reply]
I see! Thank you very much!

**Reply by EmmettPeng** [reply]
> In the context of metabolic engineering and in minimal media, the growth rate or production flux typically directly depend on the carbon source uptake rate.

@Midnighter Hi, thanks for your explanation. How can I list the summary for element like S or N?

**Reply by Midnighter** [reply]
Your question relates somewhat to https://github.com/opencobra/cobrapy/issues/1119 so at the moment you need to change the element when manually calling a display method on a `ModelSummary` instance, see https://cobrapy.readthedocs.io/en/latest/autoapi/cobra/summary/model_summary/index.html#cobra.summary.model_summary.ModelSummary.to_string and https://cobrapy.readthedocs.io/en/latest/autoapi/cobra/summary/model_summary/index.html#cobra.summary.model_summary.ModelSummary.to_html
